protocol-designer/src/ui/steps/actions/thunks/index.ts

```diff
@@ -257,3 +259,68 @@ export const saveSetTempFormWithAddedPauseUntilTemp: () => ThunkAction<any> = ()
     assert(false, 'could not auto-save pause form, getUnsavedForm returned')
   }
 }
+
+export const saveHeaterShakerFormWithAddedPauseUntilTemp: () => ThunkAction<any> = () => (
+  dispatch,
+  getState
+) => {
+  const initialState = getState()
+  const unsavedSetHeaterShakerTemperatureForm = getUnsavedForm(initialState)
```

Comment:
this is a heater shake form, so i think this should be called `unsavedHeaterShakerForm `
```suggestion
  const unsavedHeaterShakerForm = getUnsavedForm(initialState)
```